frontend/dockerfile/dockerfile_windows_test.go

```diff
@@ -0,0 +1,46 @@
+//go:build windows
+// +build windows
+
+package dockerfile
+
+import "testing"
+
+func runReproTests(t *testing.T) {
+	t.Skip("ReproTests: skipped on Windows.")
+}
+
+func runSecurityTests(t *testing.T) {
+	t.Skip("SecurityTests: skipped on Windows")
+}
+
+func runNetworkTestsWithMatrix(t *testing.T) {
+	t.Skip("SecurityTestsWithMatrix: skipped on Windows")
+}
+
+// Dockerfile cases
+var caseEnvEmptyFormatting = `
```

Comment:
These should be inlined in the actual test for readability. Otherwise hard to follow how the test works. We don't have a binary size issue for the tests so don't need to use build tags for cases where runtime condition is more readable.

Reply:
sure, makes sense. Let me know what you think about the select function I've used instead?